src/main/kotlin/step4/racingcar/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,37 @@
+package step4.racingcar.racingcar
+
+class CarRacing(private val racers: List<String>, private val moveCount: Int) {
+    private val _cars: MutableList<Car> = mutableListOf()
+
+    val gameResult: GameResult
+        get() = GameResult(this._cars, moveCount)
+
+    fun execute() {
+        this._cars.addAll(racers.map { Car(it) })
+
+        repeat(moveCount) {
+            executeCarRacing(_cars)
+        }
+    }
+
+    fun getWinner(): String {
+        val cars = this.gameResult.cars
+        val maxCount = cars.maxOf { car -> car.getMovementCount() }
+        return cars.filter { car -> car.getMovementCount() == maxCount }.joinToString { it.racerName }
+    }
+
+    private fun executeCarRacing(carList: List<Car>) {
+        carList.forEach {
+            val random = generateRandomNumber()
+            it.move(random)
+        }
+    }
+
+    private fun generateRandomNumber() = RANDOM_RANGE.random()
+    private val RANDOM_RANGE = (0..9)
```

Comment:
```suggestion
   companion object {
        private val RANDOM_RANGE = (0..9)
    }
```

class 내에서 RANDOM_RANGE를 상수로 사용하려면, companion object에 넣어야합니다.